codegen/gql_code_builder/lib/var.dart

```diff
@@ -71,10 +91,22 @@ Library buildVarLibrary(
           typeOverrides: typeOverrides,
         ),
       ),
-      hasCustomSerializer: hasCustomSerializer,
-      methods: [
-        if (hasCustomSerializer)
-          nullAwareJsonSerializerField(frag, "G${frag.name.value}Vars"),
```

Comment:
I think this was removed by mistake. 
This removes the custom serializer which can distinguish between null and absent values if the tristate value config is enabled.

Reply:
Yes!! 💯 I was initially working off of a year old branch, I'll fix it tomorrow morning